locale/zh_CN.po

```diff
@@ -1522,24 +1522,26 @@ msgid "RUN_AUTO_EVO_DURING_GAMEPLAY"
 msgstr "在游戏中启用auto-evo"
 
 #: ../src/general/OptionsMenu.tscn:810
+#, fuzzy
 msgid "DETECTED_CPU_COUNT"
-msgstr "检测到的CPU数量："
+msgstr "选中的："
```

Comment:
It seems this PR has a bunch of translation reverts?
These need to be fixed, we don't want to lose the translators work.

Reply:
I think it should be fixed now, messed it up during the previous merge due to my master branch not being up to date.